Sources/AblyAssetTrackingInternal/AblyWrapper/AblySubscriber.swift

```diff
@@ -0,0 +1,88 @@
+import Foundation
+import AblyAssetTrackingCore
+import CoreLocation
+
+public protocol AblySubscriberServiceDelegate: AnyObject {
+    /**
+     Tells the delegate that `Ably` client connection state changed.
+     
+     - Parameter sender:    The `AblySubscriber` object which is delegating the change.
+     - Parameter state:     The `ConnectionState` object
+     */
+    func subscriberService(sender: AblySubscriber, didChangeClientConnectionStatus status: ConnectionState)
```

Comment:
didChangeClientConnectionStatus: status vs state, I think they should be called identically in both delegates

Reply:
yeah, this is a copy-paste from previous delegation implementation (I didn't touch this one), but I agree -> I'll change it for the coherence